cocos/ui/scroll-view.ts

```diff
@@ -456,8 +460,9 @@ export class ScrollView extends ViewGroup {
      * ```
      */
     public scrollToBottom (timeInSecond?: number, attenuated = true) {
+        _tempAnchorVec2.set(0, 0);
         const moveDelta = this._calculateMovePercentDelta({
```

Comment:
Please modify the function to accept three parameters instead of one option parameter